src/test/com/yetanalytics/squuid_test.cljc

```diff
@@ -58,3 +61,40 @@
                       [new-u max-2]))
                   (repeatedly 30)
                   (every? (partial apply =))))))))
+
+(deftest time->uuid-test
+  (testing "Matching #inst and #uuid"
+    (let [uuid #uuid "017dfcad-ef95-8fff-8fff-ffffffffffff"
+          inst #inst "2021-12-27T16:16:37.269Z"]
+      (is (= uuid (squuid/time->uuid inst)))))
+  (testing "input must be #inst"
+    #?(:clj
+       (is (thrown? Exception
+                    (squuid/time->uuid "2021-12-27T16:16:37.269Z")))
+       :cljs
+       (is (thrown-with-msg? js/Error #"Assert failed"
+                             (squuid/time->uuid "2021-12-27T16:16:37.269Z"))))))
```

Comment:
This test will fail because instrumentation is on, so the error would be a spec error instead of an assertion error.